content/docs/reference-react-dom-server.md

```diff
@@ -17,109 +17,113 @@ var ReactDOMServer = require('react-dom/server');
 
 ## Overview {#overview}
 
-The following methods can be used in both the server and browser environments:
-
-- [`renderToString()`](#rendertostring)
-- [`renderToStaticMarkup()`](#rendertostaticmarkup)
-
-These additional methods depend on a package (`stream`) that is **only available on the server**, and won't work in the browser.
+These methods are only available in the **environments with [Node.js Streams](https://nodejs.dev/learn/nodejs-streams):**
 
 - [`renderToPipeableStream()`](#rendertopipeablestream)
-- [`renderToReadableStream()`](#rendertoreadablestream)
 - [`renderToNodeStream()`](#rendertonodestream) (Deprecated)
 - [`renderToStaticNodeStream()`](#rendertostaticnodestream)
 
-* * *
-
-## Reference {#reference}
-
-### `renderToString()` {#rendertostring}
-
-```javascript
-ReactDOMServer.renderToString(element)
-```
-
-Render a React element to its initial HTML. React will return an HTML string. You can use this method to generate HTML on the server and send the markup down on the initial request for faster page loads and to allow search engines to crawl your pages for SEO purposes.
-
-If you call [`ReactDOM.hydrateRoot()`](/docs/react-dom-client.html#hydrateroot) on a node that already has this server-rendered markup, React will preserve it and only attach event handlers, allowing you to have a very performant first-load experience.
-
-* * *
-
-### `renderToStaticMarkup()` {#rendertostaticmarkup}
+These methods are only available in the **environments with [Web Streams](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API)** (this includes browsers, Deno, and some modern edge runtimes):
 
-```javascript
-ReactDOMServer.renderToStaticMarkup(element)
-```
+- [`renderToReadableStream()`](#rendertoreadablestream)
 
-Similar to [`renderToString`](#rendertostring), except this doesn't create extra DOM attributes that React uses internally, such as `data-reactroot`. This is useful if you want to use React as a simple static page generator, as stripping away the extra attributes can save some bytes.
+The following methods can be used in both the server and browser environments:
```

Comment:
Using the term “browser” environment is misleading because we *never* recommend using these in the browser except maybe in a Service Worker for dev.

They’re only for Web-like environments that is not actually the Browser.